### PR TITLE
Make dense trio MT

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: gnomad_qc/v4/resources/sample_qc.py [hunk 1]
```diff
@@ -997,6 +997,31 @@ def ped_filter_param_json_path(
     return f"{get_sample_qc_root(version)}/relatedness/trios/gnomad.exomes.v{version}.ped_filters.json"
 
 
+def dense_trio_mt(
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/resources/sample_qc.py:1000]: Yeah, I'm going to make changes to the de novos in another PR, but yup, that's the plan

File: gnomad_qc/v4/resources/basics.py [hunk 2]
```diff
@@ -158,6 +161,12 @@ def get_gnomad_v4_vds(
             logger.info("Filtering to chromosome %s...", chrom)
             vds = hl.vds.filter_chromosomes(vds, keep=chrom)
 
+    if naive_coalesce_partitions:
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/resources/basics.py:164]: I think the naive coalesce is just loading in multiple partitions at once and then doing the filtering on the loaded partitions

File: gnomad_qc/v4/sample_qc/identify_trios.py [hunk 3]
```diff
@@ -385,12 +387,62 @@ def filter_ped(
...
+        & hl.is_defined(meta_ht[fam_ht.mat_id])
+    )
+    meta_ht = filter_to_trios(meta_ht, fam_ht)
+
+    # Get the gnomAD VDS filtered to high quality releasable trios.
+    vds = get_gnomad_v4_vds(
+        high_quality_only=True,
+        chrom="chr20" if test else None,
+        filter_samples_ht=meta_ht,
+        entries_to_keep=["LA", "LGT", "LAD", "LPGT", "LPL", "DP", "GQ", "SB"],
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/sample_qc/identify_trios.py:427]: it's basically all the entries that are not `gvcf_info`. I can clean that up if you want to get all the entry fields minus `gvcf_info`

[the remark this answers is not in the document]
Comment (KoalaQin) [reply gnomad_qc/v4/sample_qc/identify_trios.py:427]: Do you mind adding a note? I'm still not familiar what entries we have in the VDS, does it change from version to version?

**Review by KoalaQin — APPROVED** [review]
LGTM!

**Pull request merged**